internal/net/grpc/pool/pool.go

```diff
@@ -155,15 +155,15 @@ func (p *pool) Connect(ctx context.Context) (c Conn, err error) {
 			log.Debugf("establishing balanced connection to %s", addr)
 			conn, err := p.dial(ctx, addr)
 			if err != nil {
-				log.Debug(err)
+				log.Debugf("An error occurred during dialing to %s: %s", addr, err)
```

Comment:
should we use `debug` or `warn` here>

Reply:
I'm not sure but it seems an irregular situation. So I think `Warn` is better.
```suggestion
				log.Warnf("An error occurred during dialing to %s: %s", addr, err)
```